test/cmocka/src/common_mocks.c

```diff
@@ -1,14 +1,56 @@
 // SPDX-License-Identifier: BSD-3-Clause
 //
-// Copyright(c) 2019 Intel Corporation. All rights reserved.
+// Copyright(c) 2020 Intel Corporation. All rights reserved.
```

Comment:
The date should be the first time it was done (2019) or range when it was done and edited (2019-2020)

Reply:
Ok, done